.env.production

```
@@ -1 +1 @@
-VITE_API_URL=https://api.idp.gistory.me
+VITE_API_URL=https://api.idp.gistory.me
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)